planner/core/rule_column_pruning.go

```diff
@@ -53,6 +53,16 @@ func getUsedList(usedCols []*expression.Column, schema *expression.Schema) ([]bo
 	return used, nil
 }
 
+// exprsHasSideEffects checks if any the expressions has side effects.
```

Comment:
nit: `if any the expressions`  -> `if any of the expressions`

Reply:
fixed,thanks